### PR TITLE
Update webpack descriptions

### PR DESCRIPTION
**Pull request by SebastianAigner**
More detailed description around webpack and `webpack.config.d`, DSL,
Add description about .yarnrc (mostly for defining custom registries)
Remove capitalizaion from "webpack" as per branding guideline (https://webpack.js.org/branding/)

### CODE REVIEW

File: pages/docs/reference/js-project-setup.md [hunk 1]
```diff
@@ -337,18 +337,38 @@ To run tests, execute the standard lifecycle `check` task:
...
-capabilities are well described in its [documentation](https://webpack.js.org/concepts/configuration/).
-For Kotlin/JS projects, the Webpack configuration files should reside in the `webpack.config.d` directory inside the 
-root project directory.
+For browser targets, the Kotlin/JS plugin uses the widely known [webpack](https://webpack.js.org/) module bundler.
+
+The Kotlin/JS Gradle plugin automatically generates a standard webpack configuration file 
+at build time which you can find the at `build/js/packages/projectName/webpack.config.js`.
+
+The most common webpack adjustments can be made directly through the Kotlin Gradle DSL via the
+`kotlin.target.browser.webpackTask` configuration block in the `build.gradle.kts` file.
```
Comment (p7nov) [inline pages/docs/reference/js-project-setup.md:348]: Please rephrase this to apply to both Groovy and Kotlin build scripts

File: pages/docs/reference/js-project-setup.md [hunk 2]
```diff
@@ -337,18 +337,38 @@ To run tests, execute the standard lifecycle `check` task:
...
+
+If you want to make further adjustments to the webpack configuration, place your additional configuration files inside a directory
+called `webpack.config.d` in the root of your project. When building your project, all JS files will automatically
```
Comment (p7nov) [inline pages/docs/reference/js-project-setup.md:351]:
```suggestion
called `webpack.config.d` in the root of your project. When building your project, all JS configuration files will automatically
```

File: pages/docs/reference/js-project-setup.md [hunk 3]
```diff
@@ -358,6 +378,23 @@ To build a project artifact using Webpack, execute the `browserProductionWebpack
 
 </div>
 
+## Configuring yarn
```
Comment (p7nov) [inline pages/docs/reference/js-project-setup.md:381]:
```suggestion
## Configuring Yarn

```

Comment (p7nov) [reply pages/docs/reference/js-project-setup.md:381]: Unlike **webpack**, **Yarn** is always capitalized as I see in their docs :)
Please fix other occurrences.

**Comment by SebastianAigner** [comment]
Thanks for reviewing, @p7nov. I've made the changes as discussed. Feel free to merge, or let me know if you'd like additional changes.

**Comment by p7nov** [comment]
Thanks, @SebastianAigner!
I've also fixed capitalization on `npm` in this branch.
